installation/language/en-GB/joomla.cli.ini

```
@@ -0,0 +1,34 @@
+INSTL_SITE_NAME_DESC="Enter the name of your Joomla site."
+INSTL_SITE_NAME_DESC_SHORT="Name of the website"
+INSTL_ADMIN_USER_DESC="Enter the real name of your Super User. "
+INSTL_ADMIN_USER_DESC_SHORT="Real name of the Super User account"
+INSTL_ADMIN_USERNAME_DESC="Set the username for your Super User account."
+INSTL_ADMIN_USERNAME_DESC_SHORT="Username of your Super User account"
+INSTL_ADMIN_PASSWORD_DESC="Set the password for your Super User account."
+INSTL_ADMIN_PASSWORD_DESC_SHORT="Password of your Super User account"
+INSTL_ADMIN_EMAIL_DESC="Enter the email address of the website Super User."
+INSTL_ADMIN_EMAIL_DESC_SHORT="Email address of the websites Super User account"
+INSTL_DATABASE_TYPE_DESC="Database type. Supported: mysql, mysqli, pgsql, postgresql"
+INSTL_DATABASE_TYPE_DESC_SHORT="Database type. Supported: mysql, mysqli, pgsql, postgresql"
+INSTL_DATABASE_HOST_DESC="Database host"
+INSTL_DATABASE_HOST_DESC_SHORT="Database host"
+INSTL_DATABASE_USER_DESC="Database username"
+INSTL_DATABASE_USER_DESC_SHORT="Database username"
+INSTL_DATABASE_PASSWORD_DESC="Database password"
+INSTL_DATABASE_PASSWORD_DESC_SHORT="Database password"
+INSTL_DATABASE_NAME_DESC="Database name"
+INSTL_DATABASE_NAME_DESC_SHORT="Database name"
+INSTL_DATABASE_PREFIX_DESC="Prefix for the database tables"
+INSTL_DATABASE_PREFIX_DESC_SHORT="Prefix for the database tables"
+INSTL_DATABASE_ENCRYPTION_MODE_LABEL="Encryption for the connection the database. Values: 0=None, 1=One way, 2=Two way"
```

```suggestion
INSTL_DATABASE_ENCRYPTION_MODE_LABEL="Encryption for the connection to the database. Values: 0=None, 1=One way, 2=Two way"
```

not a perfect description but at least it's english